front-end/.storybook/preview-head.html

```diff
@@ -0,0 +1,6 @@
+<head>
+  <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css"
```

Comment:
This file is basically the HTML for stories in storybook.
We get bulma css from cdn and it is visible in our main app because we import it via a `link` element in `index.html`. But the storybook environment doesn't use that index.html file, so we need to have its own html file here where we can import bulma for storybook.